src/SFA.DAS.EmployerAccounts/Interfaces/IHttpService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;

```

Comment:
Should this kind of thing be in this repository? It 'feels' like the kind of thing that should be in a shared package.

This comment applies to this group of classes/interfaces.

Reply:
Think we should review this as part of refactor work?